src/Components/BrowsePage/BrowsePage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import './BrowsePage.scss'
+import { CgUserlane } from "react-icons/cg";
+import { RiHomeSmileLine } from "react-icons/ri";
+import { Link } from 'react-router-dom';
+
+export const BrowsePage = (props) => {
+    const injectActivityInformation = () => {
+        return props.data.map((e, i) => {
+            return (
+                <li key ={i}className="hexa">
+                    <div className="inner-hexagon">
+                        <img className="hexa-image" 
+                        src={e.image_poster || e.image_100 || e.image} 
+                        alt={e.song_title || e.instructions || e.podcast_title || e.name}
+                        />
+                    {e.name && <h1 className={e.name.length  < 20 ? "title" : "large-title"}>{e.name }</h1>}
+                    {e.podcast_title && <h1 className={e.podcast_title.length  < 20 ? "title": "large-title"}>{e.podcast_title }</h1>}
+                    {e.song_title && <h1 className={e.song_title.length  < 20 ? "title" : "large-title"}>{e.song_title }</h1>}
+                    {e.title && <h1 className={e.title.length  < 20 ? "title" : "large-title"}>{e.title }</h1>}
```

Comment:
Interesting use of conditionals here to determine different styles, definitely a good idea to keep thinks looking good with all the varying lengths of text